src/containers/smart-collection/store/sagas.ts

```diff
@@ -17,9 +17,9 @@ import { getAccountStatus } from 'store/account/selectors'
 import { setSmartCollection } from 'containers/collection-page/store/actions'
 import Track from 'models/Track'
 import { processAndCacheTracks } from 'store/cache/tracks/utils'
-import AudiusBackend from 'services/AudiusBackend'
 import { Status } from 'store/types'
 import { EXPLORE_PAGE } from 'utils/route'
+import { fetchRandomTracks } from '../../../store/recommendation/sagas'
```

Comment:
nit, this can be just 'store/recommendation/sagas' (absolute urls work from the project root). probably annoying since intellij won't do this by default, but maybe there's a setting?

Reply:
There's gotta be one (hopefully). Good catch!